Framework/PythonInterface/plugins/algorithms/MaskAngle.py

```diff
@@ -60,8 +60,20 @@ def validateInputs(self):
                 hasInstrument = hasInstrument and len(item.componentInfo()) > 0
         else:
             hasInstrument = len(ws.componentInfo()) > 0
+
         if not hasInstrument:
             issues["Workspace"] = "Workspace must have an associated instrument."
+
+        angleMin = self.getProperty('MinAngle').value
+        angleMax = self.getProperty('MaxAngle').value
+        if self.getProperty('Angle').value != 'InPlane':
```

Comment:
Don't do this. Force people to use positive angles, otherwise it gets confusing. Some people might interpret that masking angle from -5 to 20 will mask 0 as well.

Reply:
What is a good way to specify right and left side of the instrument separately without the negative sign?

Reply:
The min angle is fine, for in plane. Do not take the absolute value for theta or phi. Just throw an error